CHANGELOG.latest.md

```diff
@@ -1,18 +1,20 @@
-#### API updates: 
+#### API updates:
 
-- Added new method `checkTrialOrIntroDiscountEligibility(product:)`, which allows you to check for intro or discount eligibility for a single `StoreProduct`. 
-    https://github.com/RevenueCat/purchases-ios/pull/1354
+- Added new method `setMixpanelDistinctID` as a convenience method for setting the required attribute for the Mixpanel integration #1397
 
-- Added explicit parameter name for `checkTrialOrIntroDiscountEligibility(productIdentifiers:)`. 
-The existing method without the parameter name still work, but is now deprecated. Xcode will offer an auto fix-it for it. 
-    https://github.com/RevenueCat/purchases-ios/pull/1362
+- Async promotional offer methods no longer include the `get` prefix #1405
```

Comment:
I think this line is no longer needed now that we have the other two, I'd remove it so that it doesn't look like we're breaking API